scripts/postgresql-setup.sh

```diff
@@ -22,6 +22,14 @@ case "$(uname)" in
     force="-f"
   esac
 
+# Only Linux is officially supported so this should be fine.
+numcores=$(grep -c processor /proc/cpuinfo)
```

Comment:
Another way of doing this that works everywhere ( I think... tried ubuntu, rocky linux and freebsd at least ... ) is: getconf _NPROCESSORS_ONLN